desktop-widgets/templateedit.h

```diff
@@ -29,13 +32,15 @@ private slots:
 	Ui::TemplateEdit *ui;
 	QButtonGroup *btnGroup;
 	bool editingCustomColors;
-	const print_options &printOptions;
+	print_options &printOptions;
 	template_options &templateOptions;
 	struct template_options newTemplateOptions;
 	QString grantlee_template;
 	void saveSettings();
 	void updatePreview();
-
+#ifdef USE_WEBENGINE
+	Printer *preview;
```

Comment:
[3]

Reply:
Please elaborate

Reply:
This can be removed if we manage to resolve https://github.com/subsurface/subsurface/pull/3353#discussion_r756419031

Reply:
See above, I think this is better with a QWebEngineView directly.